components/meta-packages/build-essential/build-essential.p5m

```diff
@@ -43,7 +43,7 @@ depend fmri=developer/sunstudio12u1 type=require
 
 # Headers
 depend fmri=system/header type=require
-depend fmri=system/library/math/header-math type=require
+depend fmri=system/library/math type=require
```

Comment:
Please, if you add system/library/math, add it to 'Useful libraries' section. Also note that you don't need it as system/library depends on system/library/math